x-pack/plugins/session_view/public/types.ts

```diff
@@ -61,17 +61,34 @@ export interface DetailPanelProcessLeader {
   id: string;
   name: string;
   start: string;
-  end?: string;
-  exit_code?: number;
+  end: string;
+  exitCode: string;
   userName: string;
   groupName: string;
-  working_directory: string;
-  tty?: Teletype;
-  args: string[];
-  pid?: number;
+  workingDirectory: string;
+  interactive: string;
```

Comment:
should this be type 'boolean'?

Reply:
I was thinking to use the fields in this interface for the final data that will be rendered, in this case `interactive` will contain the result from https://github.com/elastic/kibana/pull/131349/files#diff-fe2484ea044c836622b7f82c2deba600dec75047e488e533f75e9300bd250179R70